docs/widget_callback_props.md

```diff
@@ -288,7 +288,6 @@ import { ConnectWidget } from "@mxenabled/react-native-widget-sdk"
 - Payload fields:
     - `user_guid` (`string`)
     - `session_guid` (`string`)
-    - `member_guid` (`string`)
```

Comment:
I didn't make these changes manually, they're updates brought in from the post message defs package.

Reply:
No need to be confused, and it's not a bug either. This was removed from the post messages defs package because in my testing I never got a `member_guid`.

Reply:
This PR adds that field back: https://github.com/mxenabled/widget-post-message-definitions/pull/14. Once merged, I'll update this PR.